supervision/detection/line_counter.py

```diff
@@ -197,6 +201,69 @@ def __init__(
         self.text_padding: int = text_padding
         self.custom_in_text: str = custom_in_text
         self.custom_out_text: str = custom_out_text
+        self.display_in_count: bool = display_in_count
+        self.display_out_count: bool = display_out_count
+
+    def _annotate_count(
+        self,
+        frame: np.ndarray,
+        line_counter: LineZone,
+        text: str,
+        in_count: bool,
+    ) -> None:
+        """This method is drawing the text on the frame.
+
+        Args:
+            frame (np.ndarray): The image on which the text will be drawn.
+            line_counter (LineCounter): The line counter
+                that will be used to draw the line.
+            text (str): The text that will be drawn.
+            in_count (bool): Whether to display the in count or not.
+        """
+        (text_width, text_height), _ = cv2.getTextSize(
+            text, cv2.FONT_HERSHEY_SIMPLEX, self.text_scale, self.text_thickness
+        )
+
+        text_x = (
```

Comment:
I think we could add vector `center` property and hide this logic there.

```python
@dataclass
class Vector:
    start: Point
    end: Point

    @property
    def center(self) -> Point:
        pass
```

Reply:
Thanks for your advice. I updated my PR.